app/(header-default)/dirigeants/[slug]/_component/liens-capitalistiques/protected-liens-capitalistiques-section.tsx

```diff
@@ -0,0 +1,184 @@
+'use client';
+
+import { Loader } from '#components-ui/loader';
+import { Select } from '#components-ui/select';
+import { DGFiP } from '#components/administrations';
+import { Section } from '#components/section';
+import { FullTable } from '#components/table/full';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { isAPI404 } from '#models/api-not-responding';
+import { ISession } from '#models/authentication/user/session';
+import { IUniteLegale } from '#models/core/types';
+import { hasAnyError, isDataLoading } from '#models/data-fetching';
+import {
+  IEtatCivilLiensCapitalistiques,
+  IPersonneMoraleLiensCapitalistiques,
+} from '#models/rne/types';
+import { UseCase } from '#models/use-cases';
+import { pluralize } from '#utils/helpers/formatting/formatting';
+import EtatCivilInfos from 'app/(header-default)/dirigeants/[slug]/_component/sections/entreprise/EtatCivilInfos';
+import PersonneMoraleInfos from 'app/(header-default)/dirigeants/[slug]/_component/sections/entreprise/PersonneMoraleInfos';
+import { APIRoutesPaths } from 'app/api/data-fetching/routes-paths';
+import { useAPIRouteData } from 'hooks/fetch/use-API-route-data';
+import { ChangeEvent, useMemo, useState } from 'react';
+
+function LiensCapitalistiquesContent({
+  uniteLegale,
+  session,
+  selectedYear,
+  useCase,
+}: {
+  uniteLegale: IUniteLegale;
+  session: ISession | null;
+  selectedYear: string;
+  useCase: UseCase;
+}) {
+  const params = useMemo(
+    () => ({
+      params: { year: selectedYear, useCase },
+    }),
+    [selectedYear, useCase]
+  );
+
+  const liensCapitalistiquesProtected = useAPIRouteData(
+    APIRoutesPaths.EspaceAgentLiensCapitalistiquesProtected,
+    uniteLegale.siren,
+    session,
+    params
+  );
+
+  if (isDataLoading(liensCapitalistiquesProtected)) {
+    return <Loader />;
+  }
+
```

Comment:
Same issue as with `<LiassesFiscales />` once a year has been selected, `<Loader />` is not displayed anymore when we change year

Reply:
> Same issue as with `<LiassesFiscales />` once a year has been selected, `<Loader />` is not displayed anymore when we change year

I've fixed it for both :)